src/deploy/osp_deployer/settings/sample_csp_profile.ini

```diff
@@ -171,6 +171,29 @@ StorageBond1Interface1=em2
 StorageBond1Interface2=p2p2
 StorageBondInterfaceOptions=mode=802.3ad miimon=100 xmit_hash_policy=layer3+4 lacp_rate=1
 
+[Dell NFV Settings]
```

Comment:
There is a second Dell NFV Settings section here:
https://github.com/dsp-jetpack/JetPack/pull/26/files#diff-03907ea6ba7811b5a8d4acb7b8c60701L255
One or the other needs to be removed.

Reply:
This also needs to go in the sample_xsp_profile.ini ( with the values disabled accordingly)